### PR TITLE
fix: Enable TLS in google cloud via cloud proxy

### PR DESCRIPTION
**Pull request by jsbroks**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -138,7 +138,7 @@ module "gke_app" {
   host                       = local.url
   bucket                     = "gs://${local.bucket}"
   bucket_queue               = local.bucket_queue
-  database_connection_string = "mysql://${module.database.connection_string}"
+  database_connection_string = "cloudsql://${module.database.connection_string}"
```
Comment (flamarion) [inline main.tf:141]: Is this tested? I have a task for this test but I didn't finish it yet, but if you did I will archive the task.

Comment (jsbroks) [reply main.tf:141]: Its not tested

Comment (flamarion) [reply main.tf:141]: I will put this in front of other tasks and get it tested with the `0.30.0`

Comment (vanpelt) [reply main.tf:141]: This won't work.  The connection_string is using the private IP address.  We need to modify the string to use the connection name with colons replaced with period.

Comment (zythosec) [reply main.tf:141]: We can do it in two steps:
1. put it behind a variable, default that variable to off as the default 
2. next release of this module, default to on and customers still have the ability to keep it off until they upgrade the server

**Pull request closed without merging**